doc/pages/Manual.md

```diff
@@ -256,46 +497,16 @@ reference name (branch, commit, HEAD...): `git://foo.com/git/bar#master`,
 The URLs given for user information (e.g. package homepages and bugtrackers) are
 not concerned and should just load nice in common browsers.
 
-## Repository format
-
-A repository is a target that can be pointed to by the `opam repository add`
-command and provides metadata (and possibly archives) to OPAM.
-
-### Layout
-
-An OPAM repository is a directory structure containing the following elements:
-
-- a `packages/` sub-directory, holding all package definitions
-- a `compilers/` sub-directory, holding the compilers definitions (used by `opam
-  init` and `opam switch`)
-- a `repo` file with some metadata on the repository itself
-- an `archives/` subdirectory that may hold source archives for the packages
-- possibly `index.tar.gz` and `urls.txt` files, when serving over HTTP.
-
-All of them are optional, although the `repo` file is recommended if the repo
-isn't for purely local use.
-
-The packages definitions are held in `package-name.package-version` directories
-below `packages/`, at any depth. The convention on the official repository is to
-have a directory per package name, gathering all versions, as such:
-`packages/package-name/package-name.package-version/`. If the directory listing
-gets too large in the future, it may be split into sub-directories by prefix
-(e.g. `packages/p/pa/package-name/package-name.version`).
+### Specifications of the formats of all files
 
-The compiler definitions are held in directories below `compilers/`, at any
-depth. The convention is to place them in separate sub-directories depending on
-the OCaml version they are based on: They should have a name of the form
-`base-ocaml-version+patch` (e.g. `4.02.1+BER` for the `BER` compiler), and be
-placed in a directory `compilers/base-ocaml-version/base-ocaml-version+patch/`.
-For the official OCaml releases, the `+patch` suffix is omitted.
-
-### Repo specification
+#### repo
+<a id="Repospecification"></a>
 
 The `repo` file, placed at the root of a repository, has the following optional
 fields:
 
 * <a id="repofield-opam-version">`opam-version: <string>`</a>:
-  File format and repository format version, should be `1.2` as of writing.
+  File format and repository format version, should be `2.0` as of writing.
 * <a id="repofield-browse">`browse: <string>`</a>:
```

Comment:
As I said in a comment I don't feel like there's the need to introduce a new name here, `homepage` or `doc` would do.